MdeModulePkg/Universal/Variable/RuntimeDxe/VariableSmmRuntimeDxe.inf

```diff
@@ -119,7 +119,8 @@
   gEdkiiVariableRuntimeCacheInfoHobGuid

```

Comment:
Bug in VariableSmmRuntimeDxe easily can be its own PR

Reply:
If this isn't blocking you, I would recommend splitting it out, since this is such a large PR.

Reply:
This sadly will cause OneCryptoPkg to fail due to the way the depex is handeling the loaders. I can split it out but it would mean I would need that to go in first.